lib/compat/wordpress-6.1/script-loader.php

```diff
@@ -156,7 +156,9 @@ function gutenberg_enqueue_global_styles() {
 	wp_enqueue_style( 'global-styles' );
 
 	// add each block as an inline css.
-	wp_add_global_styles_for_blocks();
+	if ( ! function_exists( 'wp_add_global_styles_for_blocks' ) ) {
+		wp_add_global_styles_for_blocks();
+	}
```

Comment:
This check appears to call the function only if it doesn't exist.

Instead, should we use a `gutenberg` prefixed version of the function?